executor/simple.go

```diff
@@ -1011,3 +1014,10 @@ func (e *SimpleExec) autoNewTxn() bool {
 	}
 	return false
 }
+
+func (e *SimpleExec) executeShutdown(s *ast.ShutdownStmt) error {
+	sessVars := e.ctx.GetSessionVars()
+	logutil.BgLogger().Info("execute shutdown statement", zap.Uint64("conn", sessVars.ConnectionID))
+	// sending signal to signal.SetupSignalHandler
+	return syscall.Kill(syscall.Getpid(), syscall.SIGTERM)
```

Comment:
Could we use https://godoc.org/os#Process.Kill to adapt to more platforms?

Reply:
Done

Reply:
Current server's log (we do not execute graceful shutdown):

```
[2019/10/21 10:24:18.395 +03:00] [INFO] [server.go:411] ["new connection"] [conn=1] [remoteAddr=127.0.0.1:49830]
[2019/10/21 10:24:21.370 +03:00] [INFO] [simple.go:1020] ["execute shutdown statement"] [conn=1]
[1]    20204 killed     ./bin/tidb-server
```

Current client's log:

```
mysql> shutdown;
ERROR 2006 (HY000): MySQL server has gone away
```